3.5/runtime/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM microsoft/windowsservercore
-
-ADD install /build
-RUN DISM /Online /Add-Package /PackagePath:C:\build\microsoft-windows-netfx3-ondemand-package.cab & \
-    DISM /Online /Add-Package /PackagePath:C:\build\patch\Windows10.0-KB3213986-x64.cab  & \
-    rd /s /q C:\build
-RUN set COMPLUS_NGenProtectedProcess_FeatureEnabled=0 & \
-    \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen update & \
-    \Windows\Microsoft.NET\Framework\v4.0.30319\ngen update & \
-    \Windows\Microsoft.NET\Framework64\v2.0.50727\ngen update & \
-    \Windows\Microsoft.NET\Framework\v2.0.50727\ngen update
+FROM microsoft/windowsservercore
+
+ADD install /build
+RUN DISM /Online /Add-Package /PackagePath:C:\build\microsoft-windows-netfx3-ondemand-package.cab & \
+    DISM /Online /Add-Package /PackagePath:C:\build\patch\Windows10.0-KB3213986-x64.cab  & \
+    rd /s /q C:\build
+RUN set COMPLUS_NGenProtectedProcess_FeatureEnabled=0 & \
```

Comment:
Probably the wrong PR to raise this concern, but I just noticed that we aren't checking exit codes when chaining together exe invocations.
I'm not familiar with cmd, but the PowerShell equivalent would read something like this
```Dockerfile
RUN set COMPLUS_NGenProtectedProcess_FeatureEnabled=0; \
    if($LASTEXITCODE -ne 0){ Exit $LASTEXITCODE } ; \
    & 'C:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen' update ; \
    if($LASTEXITCODE -ne 0){ Exit $LASTEXITCODE } ; \
    & 'C:\Windows\Microsoft.NET\Framework\v4.0.30319\ngen' update ; \
    if($LASTEXITCODE -ne 0){ Exit $LASTEXITCODE } ; \
    & 'C:\Windows\Microsoft.NET\Framework64\v2.0.50727\ngen' update ; \
    if($LASTEXITCODE -ne 0){ Exit $LASTEXITCODE } ; \
    & 'C:\Windows\Microsoft.NET\Framework\v2.0.50727\ngen' update ;
```

Reply:
@shirhatti - Can you log a separate issue for this?

Reply:
Filed #43